assembl/static2/js/app/pages/idea.jsx

```diff
@@ -124,21 +126,25 @@ class Idea extends React.Component {
     });
     return topPosts;
   }
+
   render() {
-    const { contentLocaleMapping, lang, ideaData, ideaWithPostsData } = this.props;
+    const { contentLocaleMapping, lang, ideaLoading, ideaWithPostsData } = this.props;
     const refetchIdea = ideaWithPostsData.refetch;
-    if (ideaData.loading) {
+    if (ideaLoading) {
       return (
         <div className="idea">
           <Loader />
         </div>
       );
     }
-    const { idea } = ideaData;
+    const { announcement, id, headerImgUrl, synthesisTitle, title } = this.props;
     const isMultiColumn = ideaWithPostsData.loading ? undefined : ideaWithPostsData.idea.messageColumns.length > 0;
     const messageColumns = ideaWithPostsData.loading ? undefined : ideaWithPostsData.idea.messageColumns;
     const childProps = {
-      idea: idea,
+      idea: {
+        id: id,
+        title: title
+      },
```

Comment:
Be careful with nested object like that, it will be a new instance each time you render.

Reply:
I used `ideaId` and `ideaTitle` at first but it requires deep changes (it was during the conflict resolution)